src/library_manager/Kconfig

```
@@ -33,4 +33,12 @@ config LIBRARY_AUTH_SUPPORT
 	  could be used if enabled.
 	  If unsure say N.
 
+config LIBRARY_DEFAULT_MODULAR
+	bool "Build LLEXT modules by default"
+	depends on LLEXT
+	help
+	  Build code, that can be built as LLEXT modules as such. Usually such
```

This description is incomplete hence inaccurate:
```suggestion
	  Build as modules the LLEXT modules that are part of the special `LIBRARY_DEFAULT_MODULAR` opt-in program. Don't change the default of the other LLEXT modules. Usually such
```